src/models/UserMapper.php

```diff
@@ -642,4 +642,14 @@ public function resetPassword($token, $password)
 
         return false;
     }
+
+
+    public function delete($user_id)
+    {
+        $sql  = "delete from user where ID = :user_id";
+        $stmt = $this->_db->prepare($sql);
+        $stmt->execute(array("user_id" => $user_id));
+
+        return true;
```

Comment:
Probably :-), again I was just basing it on the delete method in https://github.com/joindin/joindin-api/blob/master/src/models/TalkMapper.php but I'm happy to change it.